### ISSUE TITLE
Increase contrast of skip to main content link

### ISSUE THREAD

**Issue by anikbrazeau** [cds-snc/notification-api#964]
The top grey menu was flagged by tota11y as lacking contrast because of its size.

![Screen Shot 2020-06-17 at 10 06 44 AM](https://user-images.githubusercontent.com/38330843/84908365-5fefa780-b082-11ea-9673-8427cb159636.png)
